src/app/CommandInputField.tsx

```diff
@@ -24,6 +25,8 @@ export const CommandInputField = observer((props: { controller: BottomPanelContr
 
     if (e.key === "Enter") {
       e.preventDefault();
+      lastCmd.push(inputValue);
```

Comment:
Do not save the command line if it is exactly the same as the last one

Reply:
Please study the mechanism of command prompt command history.

Reply:
Applied, thank you for the advices!